<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Вводные данные</string>
     </attribute>
     <widget class="QTextEdit" name="TBReqNeed">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;1000&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-во заявок требуется</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее время обработки заявки</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее время  поступления заявки</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBAverageServiceTime">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;25&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBAdmissionTime">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;10&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Лимит очереди</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBReqLimit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;26&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>начать симуляцию</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBWorkerCount">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Количество работников</string>
      </property>
     </widget>
     <widget class="QTableView" name="TLowPriority">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>181</width>
        <height>192</height>
       </rect>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonLowPriority">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить приоритет</string>
      </property>
     </widget>
     <widget class="QTableView" name="THighPriority">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>231</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonHighPriority">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить приоритет</string>
      </property>
     </widget>
     <widget class="QScrollBar" name="ScrollBarTypeRequestBound">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>350</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCDTypeRequestBound">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>300</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Выходные данные</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Коэффициент использования системы P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее время ожидания заявки в очереди Tq</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее время пребывания заявки в системе Ts</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее по времени число требований в очереди Nq</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Среднее по времени число требований в системе Ns</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Абсолютная пропускная способность системы Ca</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBP">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBTq">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBTs">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBNq">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>80</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBNs">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBCa">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-во отказов</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="TBReqFailedCoint">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Принятые заявки</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>731</width>
        <height>321</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Принятые заявки</string>
       </attribute>
       <widget class="QTableView" name="tableProccessedRequests">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>731</width>
          <height>301</height>
         </rect>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Отказы</string>
       </attribute>
       <widget class="QTableView" name="tableFailedRequests">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>731</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTextEdit" name="TBCr">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Относительная пропускная способность системы Cr</string>
      </property>
     </widget>
     <widget class="QTableView" name="TWorkerInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Информация о рабочих</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="BarsRequests" native="true">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Графики</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1211</width>
        <height>661</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Графики распределения  случайной величины</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1031</width>
          <height>531</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>График  времени обработки</string>
         </attribute>
         <widget class="QCustomPlot" name="GraphicExpService" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1021</width>
            <height>491</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_10">
         <attribute name="title">
          <string>График времени поступления</string>
         </attribute>
         <widget class="QCustomPlot" name="GraphicExpAdmission" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1021</width>
            <height>491</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Графики плотности функции распределения</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1031</width>
          <height>531</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_14">
         <attribute name="title">
          <string>График  времени обработки</string>
         </attribute>
         <widget class="QCustomPlot" name="GraphicExpDensityService" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1021</width>
            <height>481</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_15">
         <attribute name="title">
          <string>График времени поступления</string>
         </attribute>
         <widget class="QCustomPlot" name="GraphicExpDensityAdmission" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1021</width>
            <height>491</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>График по времени числа требований в очереди </string>
       </attribute>
       <widget class="QCustomPlot" name="GraphicReqPerFix" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1021</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_18">
       <attribute name="title">
        <string>График по времени числа требований в системе</string>
       </attribute>
       <widget class="QCustomPlot" name="GraphicReqInSystemPerFix" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1021</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>График по времени среднего числа требований в очереди</string>
       </attribute>
       <widget class="QCustomPlot" name="GraphicNq" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>1021</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>График по времени среднего числа требований в системе</string>
       </attribute>
       <widget class="QCustomPlot" name="GraphicNs" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1021</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Графики по времени коэффициента использования системы</string>
       </attribute>
       <widget class="QCustomPlot" name="GraphicPPerFix" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1021</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_16">
     <attribute name="title">
      <string>Схема СМО</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>611</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>ShemeSMO.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_17">
     <attribute name="title">
      <string>Блок-схемы алгоритма</string>
     </attribute>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1161</width>
        <height>661</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Блок-схема СМО.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1245</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
